chem/module_phot_tuv.F

```diff
@@ -436,13 +437,23 @@ subroutine tuv_driver(                                                 &
                                   moist(i,kts:kte,j,p_qc), moist(i,kts:kte,j,p_qi), &
                                   moist(i,kts:kte,j,p_qs), kts, kte )
            elseif( config_flags%pht_cldfrc_opt == 2 ) then
-             if( config_flags%cu_diag == 1 ) then
-               call cldfrac_fractional( cldfrac(kts:kte), &
+             if( config_flags%cu_diag == 1 ) then 
+                 if ( config_flags%phot_blcld .and. config_flags%icloud_bl == 1 ) then
+                    call cldfrac_fractional( cldfrac(kts:kte), &
+                                        moist(i,kts:kte,j,p_qv), &
+                                        moist(i,kts:kte,j,p_qc) + gd_cloud(i,kts:kte,j),  &
+                                        moist(i,kts:kte,j,p_qi) + gd_cloud2(i,kts:kte,j) + &
+                                        bl_cld(i,kts:kte,j) + bl_cld2(i,kts:kte,j),  &
```

Comment:
@jordanschnell 
I would have thought that it would be moist(qc) + gd(qc) + bl(qc) as one arg, and moist(qi) + gd(qi) + bl(qi) as the other argument. Why is bl_cld + bl_cld2 added to moist(qi) + gd(qi)?

Reply:
@davegill , you are indeed correct. Thanks for catching this mistake. I've committed the fix.

Reply:
Wow, I feel important!